src/views/data/DataView.js

```diff
@@ -17,7 +17,7 @@ const DataView = connect(
 )(({ filter, search }) => (
     <div className='data-view'>
         <Card>
-            <ControlBar/>
+            <ControlBar filter={filter}/>
```

Comment:
You don't need to pass the filter in since you're using redux's `connect` to select it inside of the `ControlBar` component already, you're just overwriting this prop to the exact same value :P.

Reply:
Removed it. I only added that while i was trying to figure out the codebase.